.github/workflows/codeql.yml

```diff
@@ -0,0 +1,101 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "main", "external-secret-poc" ]
+  pull_request:
+    branches: [ "main", "external-secret-poc" ]
```

Comment:
@Skarlso what is this branch about?
Isn't it already merged with https://github.com/open-component-model/ocm-controller/pull/252 ?

@morri-son last updates on that branch are 2 years back...

Reply:
The yaml has been generated when switching from default to advanced (to get the recommended query suite `security-extended`in). I removed the branch now.